rust/private/rustfmt.bzl

```diff
@@ -108,6 +110,11 @@ users may tag a target with `norustfmt` to have it skipped. Additionally, genera
 source files are also ignored by this aspect.
 """,
     attrs = {
+        "_config": attr.label(
```

Comment:
I'd be inclined to make this configurable (i.e. remove the leading `_`) so that people can override it if they really want - WDYT?

Reply:
Is that possible in an aspect?

Reply:
Oh, yeah, right! Indeed!